### ISSUE TITLE
Saving data in the database

### ISSUE THREAD

**Issue by Mevrael** [salomkins/php-mood#5]
in home controller:

```php
        include '../models/ContactMessage.php';
        $cm = new ContactMessage;
        $cm->name = htmlspecialchars($name);
        $cm->email = htmlspecialchars($email);
        $cm->message = htmlspecialchars($message);
        $cm->save();
```

XSS is a browser only vulnerability and not database/file related.

What if in the future you would like to store some data and allow HTML for your models? For example -  some email templates in admin panel. In that case you will have problems decoding your encoded data back in some situations. You may forgot to do that, in some cases you may have more backslashes added.

General recommended rule to follow here is:
1. store data AS IS in a database (prevent only SQL injections before performing SQL queries, but this is not that case and there is already JSON encoding used by a Model)
2. only use specific filter functions to escape your data only when you need to render it

In other words it is better to use htmlspecialchars() in view. May be you could even write a short alias for that function, For example, `e()` is often used.
